go.mod

```diff
@@ -1,12 +1,12 @@
 module github.com/bank-vaults/vault-secrets-webhook
 
-go 1.20
+go 1.21
```

Comment:
I believe this needs to be 1.21.0 or something as of Go 1.21

https://go.dev/blog/toolchain

Reply:
Thanks, that stopped `go mod tidy` to include this toolchain line